### ISSUE TITLE
Robust DenseNet121 Models

### ISSUE THREAD

**Issue by abhi1kumar** [microsoft/robust-models-transfer#9]
Hi authors,
Thank you for releasing your code and your pre-trained models. 

[Standard Accuracy](https://github.com/microsoft/robust-models-transfer#standard-accuracy-of-l2-robust-imagenet-models) Section of README benchmarks the performance of DenseNet161 on Imagenet. (The number of convolution channels for conv0 is 96 for DenseNet161 while it is 64 for DenseNet121).

Will it be possible for you to release the adversarially trained DenseNet121 model?  If your answer is No, could you please tell me which command to use to obtain an adversarially trained DenseNet121 ImageNet model?

**Comment by Hadisalman** [comment]
Hi @abhi1kumar. Apologies for the late reply. Unfortunately we don't have that model trained. If you would like to train it, you should:
1- Add an entry for the pytorch pretrained model here https://github.com/microsoft/robust-models-transfer/blob/cea2c37f09ae24485b7e593a470aa350c3fb32ee/src/main.py#L268
Specifically, add this line to the dictionary
`'densenet121': models.densenet121,`

2- Follow [this part of the readme](https://github.com/microsoft/robust-models-transfer/tree/cea2c37f09ae24485b7e593a470aa350c3fb32ee#running-transfer-learning-experiments)  with `--arch densenet121`

Hope this helps. Thanks!
